docs/chases/editing-a-chase.md

```diff
@@ -53,14 +54,14 @@ times will operate as programmed)*
     new cue is inserted.
 
 -   To **insert** a new cue at the end of the chase, set up the look for the
-    new cue, press \[Insert Step\], then press the **swop** button of the
+    new cue, press \[Insert Step\], then press the **select** button of the
```

Comment:
idk if you were going for Select always being capitalised? very nit-picky I know :P

Reply:
(I think there might be a couple of other places if you were particularly keen, obviously low stress though!)